tutormfe_extensions/plugin.py

```diff
@@ -42,7 +41,60 @@
         # Danger zone!
         # Add values to override settings from Tutor core or other plugins here.
         # Each override is a pair: (setting_name, new_value). For example:
-        ### ("PLATFORM_NAME", "My platform"),
+        (
+            "MFE_LEARNING_MFE_APP",
+            {
+                "name": "learning",
+                "repository": "https://github.com/eduNEXT/frontend-app-learning",
+                "port": 2000,
+                "version": "ednx-release/olmo.master",
+            },
+        ),
+        (
+            "MFE_ACCOUNT_MFE_APP",
+            {
+                "name": "account",
+                "repository": "https://github.com/eduNEXT/frontend-app-account",
+                "port": 1997,
+                "version": "ednx-release/olmo.master",
+            },
+        ),
+        (
+            "MFE_AUTHN_MFE_APP",
+            {
+                "name": "authn",
+                "repository": "https://github.com/eduNEXT/frontend-app-authn",
+                "port": 1999,
+                "version": "ednx-release/olmo.master",
+            },
+        ),
+        (
+            "MFE_DISCUSSIONS_MFE_APP",
+            {
+                "name": "discussions",
+                "repository": "https://github.com/eduNEXT/frontend-app-discussions",
+                "port": 2002,
+                "version": "ednx-release/olmo.master",
+            },
+        ),
+        (
+            "MFE_GRADEBOOK_MFE_APP",
+            {
+                "name": "gradebook",
+                "repository": "https://github.com/eduNEXT/frontend-app-gradebook",
+                "port": 1994,
+                "version": "ednx-release/olmo.master",
+            },
+        ),
+        (
+            "MFE_PROFILE_MFE_APP",
```

Comment:
We mentioned ora and communications in the readme, so maybe we should add them here.